src/js/console.js

```diff
@@ -48,20 +48,12 @@ function stderr(text) {
 Console.prototype.log =
 Console.prototype.debug =
 Console.prototype.info = function() {
-  if (arguments.length === 1) {
-    stdout(arguments[0] + '\n');
-  } else {
-    stdout(util.format.apply(this, arguments) + '\n');
-  }
+  stdout(util.format.apply(this, arguments) + '\n');
```

Comment:
See #140 for details, this check is for optimizing for the single argument prints.